spec/fieldhand/paginator_spec.rb

```diff
@@ -11,7 +11,7 @@ module Fieldhand
       it 'raises a Bad Argument Error if returned from the repository' do
         stub_oai_request('http://www.example.com/oai?verb=Identify&bad=Argument',
                          'bad_argument_error.xml')
-        paginator = described_class.new('http://www.example.com/oai')
+        paginator = described_class.new('http://www.example.com/oai', retries: 0)
```

Comment:
shall we make `0` as default value for retries to be fully backward compatible. So no defining anything leaves the code as it was before this change.

Reply:
This is a very good point, yes 👍